src/Magnum/ImGuiIntegration/Integration.h

```diff
@@ -0,0 +1,197 @@
+#ifndef Magnum_ImGuiIntegration_Integration_h
+#define Magnum_ImGuiIntegration_Integration_h
+/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018
+              Vladimír Vondruš <mosra@centrum.cz>
+    Copyright © 2018 ShaddyAQN <ShaddyAQN@gmail.com>
+    Copyright © 2018 lecopivo <skrivantomas@seznam.cz>
```

Comment:
`Copyright © 2018 Tomáš Skřivan <skrivantomas@seznam.cz>`  ;)

... and similarly elsewhere (if applicable)

Reply:
Oh, yeah, I think I fixed that _somewhere_ but aparently missed all other places :D